tests/handlebars-concat-statement/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -122,3 +110,54 @@ singleQuote: true
 <div class='class-p  class-q'></div>
 ================================================================================
 `;
+
+exports[`in-attr-node.hbs 1`] = `
+====================================options=====================================
+parsers: ["glimmer"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+<div class="a very long list of classes that exceeds {{eighty}} chars with emtpy spaces">hey</div>
+<a href="a-very-long-href-from-a-third-party-marketing-platform{{id}}longer-than-eighty-chars">Link</a>
+
+=====================================output=====================================
+<div
+  class="a very long list of classes that exceeds {{eighty
+    }} chars with emtpy spaces"
+>
+  hey
+</div>
+<a
+  href="a-very-long-href-from-a-third-party-marketing-platform{{id
+    }}longer-than-eighty-chars"
+>
+  Link
+</a>
+================================================================================
+`;
+
+exports[`in-attr-node.hbs 2`] = `
+====================================options=====================================
+parsers: ["glimmer"]
+printWidth: 80
+singleQuote: true
+                                                                                | printWidth
+=====================================input======================================
+<div class="a very long list of classes that exceeds {{eighty}} chars with emtpy spaces">hey</div>
+<a href="a-very-long-href-from-a-third-party-marketing-platform{{id}}longer-than-eighty-chars">Link</a>
+
+=====================================output=====================================
+<div
+  class="a very long list of classes that exceeds {{eighty
+    }} chars with emtpy spaces"
+>
+  hey
+</div>
+<a
+  href="a-very-long-href-from-a-third-party-marketing-platform{{id
+    }}longer-than-eighty-chars"
```

Comment:
I think we should avoid moving on next line in this case, it is ugly

Reply:
👍 I will update, thanks for the feedback

Reply:
Will be fixed once https://github.com/prettier/prettier/pull/7052 is rebased

Reply:
No wait, it won't. I will have to work a bit more on this issue.

Reply:
It should be good now. But we need to settle #7052 before.